src/main/java/com/discordbolt/api/command/CommandManager.java

```diff
@@ -88,8 +97,13 @@ public char getCommandPrefix(IGuild guild) {
      * @param guild         Guild to change the prefix for
      * @param commandPrefix new prefix characters all commands must be prefaced with
      */
-    public void setCommandPrefix(IGuild guild, char commandPrefix) {
-        commandPrefixes.put(guild.getLongID(), commandPrefix);
+    public void setCommandPrefix(IGuild guild, String commandPrefix) {
+        // Check prefix for space
+        if(commandPrefix.contains(" ")) {
+            logger.error("Cannot set guild prefix. Prefix cannot contain a space.");
```

Comment:
Be more specific with your error logging, something like below is much better.

`logger.error("Error while setting command prefix of guild ID '" + guild.getStringID() + "', prefix cannot contain a space.");`